server/src/main/kotlin/com/tilbox/api/user/application/EmailUserCreateService.kt

```diff
@@ -0,0 +1,42 @@
+package com.tilbox.api.user.application
+
+import com.tilbox.core.user.domain.Password
+import com.tilbox.core.user.domain.PasswordEncodingStrategy
+import com.tilbox.core.user.domain.Profile
+import com.tilbox.core.user.domain.RegistrationType
+import com.tilbox.core.user.domain.User
+import com.tilbox.core.user.domain.UserRepository
+import com.tilbox.core.user.domain.UserRole
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.time.LocalDateTime
+
+@Transactional
+@Service
+class EmailUserCreateService(
+    private val userRepository: UserRepository,
+    private val passwordEncodingStrategy: PasswordEncodingStrategy,
+) {
+    fun createUser(request: UserCreateRequest): UserCreateResponse {
+        check(!userRepository.existsByEmailAndRegistrationType(request.email, RegistrationType.EMAIL)) {
+            "이미 가입된 이메일입니다."
+        }
+        check(!userRepository.existsByMyTilAddress(request.myTilAddress)) { "이미 사용중인 TIL 주소입니다." }
```

Comment:
```suggestion
        check(!userRepository.existsByEmailAndRegistrationType(request.email, RegistrationType.EMAIL)) {
            "이미 가입된 이메일입니다. email=${request.email}"
        }
        check(!userRepository.existsByMyTilAddress(request.myTilAddress)) { "이미 사용중인 TIL 주소입니다. myTilAddress=${request.myTilAddress}"}
```
다른 곳은 이런 식으로 명시해서 메시지를 주고 계시는 데 이곳에 없어서 리뷰 남겨봅니다.

Reply:
피드백 반영했습니다!